fuel-core/src/schema/dap.rs

```diff
@@ -423,9 +443,10 @@ mod gql_types {
         Breakpoint,
     }
 
-    #[derive(Debug, Clone, Copy, SimpleObject)]
+    #[derive(Debug, Clone, SimpleObject)]
     pub struct RunResult {
         pub state: RunState,
         pub breakpoint: Option<OutputBreakpoint>,
+        pub json_receipts: Vec<String>,
```

Comment:
[nit] You could try using the graphql types instead of json serialization. Although we'd need to move the corresponding transparent_receipt in the fuel-gql-client out of the test module.

```suggestion
        pub json_receipts: Vec<crate::schema::tx::receipt::Receipt>,
```

Reply:
I think this is good enough for now